tests/Benchmarks/System.Text.Json/JsonReaderPerf.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using BenchmarkDotNet.Attributes;
+using Benchmarks;
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.IO;
+
+using static System.Text.Json.Benchmarks.Helper;
+
+namespace System.Text.Json.Benchmarks
```

Comment:
FYI, we decided on this convention for the namespace (see https://github.com/dotnet/corefxlab/pull/2248#discussion_r185659901).

cc @GrabYourPitchforks